sentinel-adapter/sentinel-motan-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/motan/BaseMotanSentinelFilter.java

```diff
@@ -0,0 +1,34 @@
+package com.alibaba.csp.sentinel.adapter.motan;
+
+import com.alibaba.csp.sentinel.adapter.motan.config.MotanAdapterGlobalConfig;
+import com.weibo.api.motan.core.extension.Activation;
+import com.weibo.api.motan.core.extension.SpiMeta;
+import com.weibo.api.motan.filter.Filter;
+import com.weibo.api.motan.rpc.Caller;
+import com.weibo.api.motan.rpc.Request;
+
+/**
+ * author: zhangxn8
+ */
+@SpiMeta(name = MotanAdapterGlobalConfig.BASE_SENTINEL_MOTAN_FILTER)
+@Activation(sequence = 1)
+public abstract class BaseMotanSentinelFilter implements Filter {
```

Comment:
Why this class is needed? Actually the implementation of the two methods is in `MotanUtils`.

Reply:
ok,  I also think is have a little redundance